packages/orchestrator-ui-components/src/hooks/ProcessesHooks/useMutateProcess.ts

```diff
@@ -0,0 +1,68 @@
+import { useContext } from 'react';
+import { useMutation, useQueryClient } from 'react-query';
+
+import { signOut } from 'next-auth/react';
+
+import { OrchestratorConfigContext } from '../../contexts';
+import { useSessionWithToken } from '../useSessionWithToken';
+
+export const useMutateProcess = () => {
+    const { processesEndpoint } = useContext(OrchestratorConfigContext);
+    const { session } = useSessionWithToken();
+    const queryClient = useQueryClient();
+
+    const genericRequestHeaders = {
+        authorization: session ? `Bearer ${session.accessToken}` : '',
+    };
+
+    const retryProcessServiceCall = async (id: string) =>
+        await fetch(`${processesEndpoint}/${id}/resume`, {
+            method: 'PUT',
+            headers: {
+                ...genericRequestHeaders,
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({}),
```

Comment:
Needed? (ignore the other comment about it when it's needed...)

Reply:
changing it into "{}"